docs/docs/intro.md

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 1
+---
+
+# Introduction
+
+This is the documentation for the *Unreal Engine* template.
```

Comment:
The introduction to what this template does/is capable of could be painted with broader strokes.

Reply:
I'd like to keep it simple for now. We can add more in the next weeks.